### PR TITLE
Add the request portal access form

### LINKED ISSUE #201: Partners link to partners portal - Build the gain access form

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add the marketo request portal access form.

## QA
- Go to /request-login
- Fill in the form
- Check the validation does its thing
- When submitted see the thank you is displayed

## Issue / Card
Fixes #201 

## Screenshots
screenshot_2019-02-20 request login ubuntu partners 2
screenshot_2019-02-20 request login ubuntu partners 1
screenshot_2019-02-20 request login ubuntu partners

### CODE REVIEW

**Comment by pmahnke** [comment]
the only stange thing, which I assume lucia did, is that is uses the comment from user to get the partner name, which gives you a textarea, not a text field... you might want to check with her that she intended that.

**Comment by pmahnke** [comment]
@anthonydillon 	added a minor change to get rid of the django error log error

**Comment by Lewciie** [comment]
Im unsure of whats the difference between a textarea and a textfield, but we do not have a "partner name" field and the best way to do it was via comments from lead field, which I had given this suggestion before to Bradley (I got no answer at the time, but it would take longer to get a new field created in SFDC).

Do you see any issues with using this field?

File: templates/request-login.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <div class="row">
+    <div class="col-8">
+      <h1>Request access to Ubuntu's partner portal</h1>
+      <p>If you need to gain access to the Ubuntu's partner portal, please fill in your details below and a member of our team will get in touch.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <script src="//app-sjg.marketo.com/js/forms2/js/forms2.min.js"></script>
+      <form id="mktoForm_3339"></form>
```
Comment (pmahnke) [inline templates/request-login.html]: could we add a fieldset?

**Comment by anasereijo** [comment]
@anthonydillon 

- [ ] Thank you page: can you replace the aubergine pictogram with [this one](https://assets.ubuntu.com/v1/cbae9a60-thank_you_orange_cmyk.svg)?

- [ ] asterisks to be grey

**Comment by anthonydillon** [comment]
@anasereijo the asterisk is red in Vanilla. https://github.com/vanilla-framework/vanilla-framework/blob/8e2005cf5d6a037038aadde7ff15005acd68d464/scss/_base_forms.scss#L151

**Comment by anasereijo** [comment]
@anthonydillon sorry I wasn't aware of that. 👍 good to go then

**Pull request merged**